### ISSUE TITLE
Support tabs for Roles

### ISSUE THREAD

**Issue by JulianSchneider** [novafloss/pydocusign#105]
According to this [documentation](https://www.docusign.com/developer-center/explore/features/templates), a role can transmit extra tabs to file additional fields.

```
 "templateRoles": [{
    "email": "john.doe@email.com",
    "name": "John Doe",
    "roleName": "Customer",
    "tabs": {
      "textTabs": [{
        "tabLabel": "CustomerAddress",
        "value": "123 Main St. San Francisco, CA 94105"
      }, {
        "tabLabel": "CustomerSSN",
        "value": "12-345-6789"
      }]
    }
  }],
```
But the Role object only supports a limited amount of fields:

```
attributes = ['clientUserId', 'email', 'emailBody', 'emailSubject', 'name',
                  'supportedLanguage', 'roleName']

```
Am I missing something or is the support not implemented?

**Comment by zebuline** [comment]
For the moment nobody needed this fonctionality, but you can propose a pull request if you want.

**Comment by zebuline** [comment]
fixes by #108/#107
